napari/utils/key_bindings.py

```diff
@@ -40,8 +40,10 @@ def hello_world(viewer):
 from types import MethodType
 from typing import Callable, Mapping, Union
 
-from app_model.types import KeyBinding, KeyCode, KeyMod
-from vispy.util import keys
+from app_model.backends.qt import qkey2modelkey, qmods2modelmods
+from app_model.types import KeyBinding, KeyCode
+from qtpy.QtCore import Qt
+from qtpy.QtGui import QKeyEvent
```

Comment:
This fails on headless as qtpy is not installed. I think one of the goal of those bindings were to be qt independant.